code/modules/research/protolathe.dm

```diff
@@ -106,6 +105,8 @@ Note: Must be placed west/left of and R&D console to function.
 		if(!in_range(src, stack) || !user.Adjacent(src))
 			return
 		var/amount_inserted = materials.insert_stack(O,amount)
+		if (amount_inserted <= 0)

```

Comment:
Go to protolathe.
Get any type of resource that can be put in the protolathe.
Place it in the protolathe.
When it gives you the pop up, type in a negative number.
Check materials storage.